utils/context_resolutor.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * A function that receives a string with a dot key foo.v1.v2
+ * and searches in the github context of the Action for its value. 
+ *
+ * If nothing is found it throws an Exception. 
+ *
+ * Args:
+ *   - key(string): a dotted key to search for its value
+ * Returns:
+ *   - value(string): the correspondant value
+ */
+
+const github = require('@actions/github');
+
+module.exports = function(key){
+
+  let resolved = false
+
+  for(const level of key.split(/\./)){
+
+    if(!resolutor){
+
+      resolved = __getResolutor(level)
+    
+    }
+    else{
+    
+      if(!resolved[l]){
+
+        throw `COULD NOT RESOLVE ${key}`
```

Comment:
```suggestion
        throw `COULD NOT RESOLVE ${{key}}`
```

Reply:
No @alambike this is a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).